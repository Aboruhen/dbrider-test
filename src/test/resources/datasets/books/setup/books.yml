book:
  - name: Ukraine
    title: UA
    id: 3
  - name: Poland
    title: PL
    id: 4
  - name: Lithuania
    title: LT
    id: 5