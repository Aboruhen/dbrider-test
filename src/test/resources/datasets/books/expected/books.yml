book:
  - name: Ukraine
    title: UA
    id: 3
  - name: groovy:(value != null && value.contains('Poland'))
    title: PL
    id: 4
  - name: Lithuania
    title: LT
    id: 5